core/encoding.go

```diff
@@ -10,19 +10,13 @@ import (
 // Commitments
 
 // Commitment is a polynomial commitment (e.g. a kzg commitment)
-type Commitment struct {
-	*bn254.G1Point
-}
+type G1Commitment bn254.G1Point
```

Comment:
`G1Commitment` & `G2Commitment` are typed as `G1Point` and `G2Point`. 
They're not being type aliased so that we can add instance methods such as `Serialize` and `Deserialize`.